guide/13-managing-arcgis-applications/hub-premium-guide.ipynb

```diff
@@ -0,0 +1,471 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "723532c9",
+   "metadata": {},
+   "source": [
+    "Next, we will connect to the Hub associated with this Web GIS."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "97c78b32",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "hub_premium = gis.hub"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "6d98bf5d",
+   "metadata": {},
+   "source": [
+    "### Adding new Initiatives\n",
+    "\n",
+    "An Initiative supports policy or activity oriented goals through workflows, tools and team collaboration. Initiatives manage the team, events, content, and experiences involved in achieving a shared goal.\n",
+    "\n",
+    "You can add a new initiative with the following commands:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "28ae447b",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/html": [
+       "<div class=\"item_container\" style=\"height: auto; overflow: hidden; border: 1px solid #cfcfcf; border-radius: 2px; background: #f6fafa; line-height: 1.21429em; padding: 10px;\">\n",
+       "                    <div class=\"item_left\" style=\"width: 210px; float: left;\">\n",
+       "                       <a href='https://dcdev.maps.arcgis.com/home/item.html?id=5ffb2e4d30244d78b19eda1c84d63db1' target='_blank'>\n",
+       "                        <img src='http://static.arcgis.com/images/desktopapp.png' class=\"itemThumbnail\">\n",
+       "                       </a>\n",
+       "                    </div>\n",
+       "\n",
+       "                    <div class=\"item_right\"     style=\"float: none; width: auto; overflow: hidden;\">\n",
+       "                        <a href='https://dcdev.maps.arcgis.com/home/item.html?id=5ffb2e4d30244d78b19eda1c84d63db1' target='_blank'><b>Initiative with Python</b>\n",
+       "                        </a>\n",
+       "                        <br/>Create your own initiative by combining existing applications with a custom site. Use this initiative to form teams around a problem and invite your community to participate.<img src='https://dcdev.maps.arcgis.com/home/js/jsapi/esri/css/images/item_type_icons/layers16.png' style=\"vertical-align:middle;\">Hub Initiative by mmajumdar_dcdev\n",
+       "                        <br/>Last Modified: December 12, 2022\n",
+       "                        <br/>0 comments, 0 views\n",
+       "                    </div>\n",
+       "                </div>\n",
+       "                "
+      ],
+      "text/plain": [
+       "<Item title:\"Initiative with Python\" type:Hub Initiative owner:mmajumdar_dcdev>"
+      ]
+     },
+     "execution_count": 4,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "initiative1 = hub_premium.initiatives.add(title='Initiative with Python')\n",
+    "initiative1.item"
+   ]
+  },
+  {
```

Comment:
I think it would help to outline here what happens when an _Initiative_ is added, or link to the documentation that outlines what happens when an _Initiative_ is created, something like:

- _Adding an `initiative` will add a Hub Application item, a Hub Initiative item, and two supporting groups, `<initiative title> Core Team` to manage members with access to the initialive, and `<initiative title> Content` to administer the content relevant to the initiative._